docs/UserGuide.md

```diff
@@ -118,7 +118,7 @@ When going through the onboarding guide, instructions such as the one shown belo
 
 ![](images/OnboardingInstruction.jpg)
 
-#### Basic features covered
+Basic features covered
```

Comment:
Likewise, you should bold the next to make it look like a heading.

Reply:
Fixed